avocado/plugins/variants.py

```diff
@@ -65,6 +66,8 @@ def configure(self, parser):
         parser.add_argument('-c', '--contents', action='store_true',
                             default=False, help="[obsoleted by --variants] "
                             "Shows the node content (variables)")
+        parser.add_argument('--dump-variants', default=None,
+                            help="Dump the Variants to a JSON file")
```

Comment:
It'd be nice to have the `--load-variants` here as well to be able to list the dumped variants other way than just with `--dry-run`.